src/views/TabView/TabBarBottom.js

```diff
@@ -71,10 +71,18 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
       inputRange,
       outputRange,
     });
+    const label = this.props.getLabelText(scene);
+    if (typeof label === 'string') {
+      return (
+        <Animated.Text style={[styles.label, { color }]}>
+          {label}
+        </Animated.Text>
+      );
+    }
     return (
```

Comment:
Not sure what you mean, nothing changes when it's string,

```js
    if (typeof label === 'string') {
      return (
        <Animated.Text style={[styles.label, { color }]}>
          {label}
        </Animated.Text>
      );
    }
    return label;
```

When you're specifying a component, we don't need to wrap it in an `Animated.View`, it doesn't serve any purpose. Also `labelStyle` is only valid on `Text` elements and you'll get warnings when you pass to `Animated.View`. And `labelStyle` is the styles passed by the user, not the default styles.